nbconvert/preprocessors/regexremove.py

```diff
@@ -20,16 +20,14 @@ class RegexRemovePreprocessor(Preprocessor):
     of unicode strings. If the contents match any of the patterns, the cell
     is removed from the notebook.
 
-    By default, `patterns = [r'\Z']` which matches the empty string such that
-    strictly empty cells are removed. To modify the list of matched patterns,
+    To modify the list of matched patterns,
     modify the patterns traitlet. For example, execute the following command
-    to convert a notebook to html and remove cells containing only whitespace:
+    to convert a notebook to html and remove cells containing only whitespace::
 
-    > jupyter nbconvert --RegexRemovePreprocessor.enabled=True \
-      --RegexRemovePreprocessor.patterns="['\\s*\\Z']" mynotebook.ipynb
+      jupyter nbconvert --RegexRemovePreprocessor.patterns="['\\s*\\Z']" mynotebook.ipynb
 
-    The first command line argument enables the preprocessor and the second
-    sets the list of patterns to '\\s*\\Z' which matches an arbitrary number
+    The command line argument
+    sets the list of patterns to ``'\\s*\\Z'`` which matches an arbitrary number
```

Comment:
Are single back ticks sufficient here?

Reply:
I tried it, and apparently not.

Sphinx allows to customize the formatting of single backticks, but it is apparently not configured for typewriter text (and I also don't think it would be a good idea to do so).